vitess.crd.yaml

```diff
@@ -0,0 +1,154 @@
+---
+apiVersion: vitess.io/v1alpha2
+kind: VitessCluster
+metadata:
+  name: superawesomecluster
+  labels:
+    app: vitess    
+spec:
+  lockserver:
+    type: etcd3
+    address: etcd-cluster-client:2379
+    path: /vitess/global
+  cells:
+  cellSelector:
+    matchLabels:
+    matchExpression:
+  keyspaces:    
+  keyspaceSelector:
+    matchLabels:
+    matchExpression:
+---
+apiVersion: vitess.io/v1alpha2
+kind: VitessCell
+metadata:
+  name: uswest
+  labels:
+    app: vitess
+spec:
+  lockserver:
+    type: etcd3
+    address: etcd-cluster-client:2379
+    path: /vitess/uswest
+  vtgate:
+    - count:
+      containers:
+        vtgate:
+          image:
+          resources:
+        ...
+      affinity:
+        ...
+      credentials:
+        secret:
+          name:
+          key:
+      cells:
```

Comment:
Where is there a cell list / cellSelectorinside of the cell resources themselves?